src/test/java/com/uid2/admin/vertx/EncryptionKeyServiceKeysetsDisabledTest.java

```diff
@@ -193,18 +193,12 @@ void rotateMasterKey(Vertx vertx, VertxTestContext testContext) throws Exception
         };
         setEncryptionKeys(MAX_KEY_ID, keys);
 
-        post(vertx, "api/key/rotate_master?min_age_seconds=100", "", ar -> {
-            assertTrue(ar.succeeded());
-            HttpResponse response = ar.result();
+        post(vertx, testContext, "api/key/rotate_master?min_age_seconds=100", "", response -> {
             assertEquals(200, response.statusCode());
             checkRotatedKeyResponse(MAX_KEY_ID+1, new int[] { -1 },
                     MASTER_KEY_ACTIVATES_IN_SECONDS, MASTER_KEY_EXPIRES_AFTER_SECONDS,
                     response.bodyAsJsonArray().stream().toArray());
-            try {
-                verify(encryptionKeyStoreWriter, times(1)).upload(collectionOfSize(3), eq(MAX_KEY_ID+1));
-            } catch (Exception ex) {
-                fail(ex);
-            }
+            verify(encryptionKeyStoreWriter, times(1)).upload(collectionOfSize(3), eq(MAX_KEY_ID+1));
             testContext.completeNow();
         });
         verifyNoInteractions(keysetStoreWriter);
```

Comment:
These calls should be inside the callback, there are a few other similar cases in this file.

Reply:
Good catch, I've updated them all